code/client/munkilib/pkgutils.py

```diff
@@ -518,7 +518,8 @@ def getFlatPackageInfo(pkgpath):
 
         productversion = None
         for toc_entry in [item for item in toc
-                          if item.startswith('Distribution')]:
```

Comment:
I do wonder why I decided to do `startswith` instead of just ==; I'm thinking maybe I've seen packages where the Distribution file was named "Distribution.xml"

Reply:
I'd love it if you or other admins could replicate my findings, but if they hold, would then advocate for a much simpler check here.

Reply:
See https://groups.google.com/g/munki-discuss/c/ilBIqbxyrfA/m/LVg_7ICbBQAJ?utm_medium=email&utm_source=footer